akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala

```diff
@@ -522,6 +523,113 @@ object StyleGuideDocExamples {
     }
   }
 
+  object BehaviorCompositionWithPartialFunction {
+
+    sealed trait Command
+    case object Down extends Command
+    final case class GetValue(replyTo: ActorRef[Value]) extends Command
+    final case class Value(n: Int)
+
+    //#get-handler-partial
+    def getHandler(value: Int): PartialFunction[Command, Behavior[Command]] = {
+      case GetValue(replyTo) =>
+        replyTo ! Value(value)
+        Behaviors.same
+    }
+    //#get-handler-partial
+
+    //#set-handler-non-zero-partial
+    def setHandlerNotZero(value: Int): PartialFunction[Command, Behavior[Command]] = {
+      case Down =>
+        if (value == 1)
+          zero
+        else
+          nonZero(value - 1)
+    }
+    //#set-handler-non-zero-partial
+
+    //#handle-partial
+    def handle(command: Command, handlers: List[PartialFunction[Command, Behavior[Command]]]): Behavior[Command] = {
```

Comment:
I think it'd be cleaner if this one didn't try to generalize it into a list of partials but rather be `a.orElse(b)` as that is what's actually needed here and in most cases where you have a state specific PF and something general to fallback to. 

If someone wants to make that more general they'll figure that out anyway.

Reply:
I will remove this and directly use `orElse`